andela/activo-web
# #158192060 Fix relative path to the font styles

#### What does this PR do?
Fix the relative paths to the font styles

#### Description of the task to be completed?
- Change the path to the fonts in globals.scss from `../../assets/fonts/**` to `../fonts/**`

#### How should this be manually tested?
- After cloning the repo, start the application using `yarn start:dev`.
- Application should be up and running on `activo-dev.andela.com/3000`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?

[#158192060](https://www.pivotaltracker.com/story/show/158192060)
